analytics/filesystem/file_module.go

```diff
@@ -32,6 +33,15 @@ func (f *FileLogger) LogAuctionObject(ao *analytics.AuctionObject) {
 	f.Logger.Flush()
 }
 
+//Writes VideoObject to file
+func (f *FileLogger) LogVideoObject(vo *analytics.VideoObject) {
+	//Code to parse the object and log in a way required
+	var b bytes.Buffer
+	b.WriteString(jsonifyVideoObject(vo))
+	f.Logger.Debug(b.String())
```

Comment:
I'm probably not understanding the whole context but if `jsonifyVideoObject` finds an error, will just flush to the log with a `DEBUG` flag?
```
"2019/12/18 14:16:03 DEBUG Transactional Logs Error: Auction object badly formed JSON_MARSHALL_ERR_MESSAGE"
```
Is this what we want? Don't we want to log to the `Error` level?

Reply:
This function is intended to log the video object at the debug level, but if there is an error in that process then yes I agree it would be better to log that as an error instead. The point of this PR however is really just to add the custom JSON request and response used by the /video endpoint to the analytics logging, so I'd prefer to leave improvements to the existing testing/logging as a separate issue. (note while this specific function to log the video object is technically new code, I say "existing" here because it's basically just a cut-&-paste of how all the other objects are being logged).